src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -115,17 +115,15 @@ class AbstractIntegrationDetailedView<
   getIcon(title: string) {
     switch (title) {
       case 'View Source':
-        return <StyledIconCode />;
+        return <IconProject />;
       case 'Report Issue':
-        return <StyledIconGithub />;
+        return <IconGithub />;
       case 'Documentation':
-        return <StyledIconDocs />;
```

Comment:
@EvanPurkhiser I am glad you cleaned up this case statement because it was bothering me lol